### PR TITLE
fix version validation 

### PR DESCRIPTION
**Pull request by Nicole00**
fix some validation and assertion

### CODE REVIEW

File: README_CN.md [hunk 1]
```diff
@@ -305,19 +320,26 @@ df = spark.read.format(
...
-|          2.0.0                 |  2.0.0, 2.0.1  |
-|          2.0.1                 |  2.0.0, 2.0.1  |
-|          2.1.0                 |  2.0.0, 2.0.1  |
-|          2.5.0                 |  2.5.0, 2.5.1  |
-|          2.5.1                 |  2.5.0, 2.5.1  |
-|          2.6.0                 |  2.6.0, 2.6.1  |
-|          2.6.1                 |  2.6.0, 2.6.1  |
-|          3.0.0                 |     3.0.0      |
-|        3.0-SNAPSHOT            |     nightly    |
+Nebula Spark Connector 和 Nebula 、Spark 的版本对应关系如下:
```
Comment (wey-gu) [inline README_CN.md:323]:
```diff
- Nebula
+ NebulaGraph
```
? 下同

File: README.md [hunk 2]
```diff
@@ -309,19 +325,25 @@ df = spark.read.format(
 
 ## Version match
```
Comment (wey-gu) [inline README.md:326]: rename to `Compatibility matrix`?

File: README_CN.md (hunk 1 above)
Comment (Nicole00) [reply README_CN.md:323]: > ```diff
> - Nebula
> + NebulaGraph
> ```
> 
> ? 下同

good catch

File: README.md (hunk 2 above)
Comment (Nicole00) [reply README.md]: got it

**Review by wey-gu — APPROVED** [review]
great job, thanks!

**Pull request merged**